src/components/Articles.jsx

```diff
@@ -0,0 +1,30 @@
+import getArticles from "../api/getArticles";
+import { useEffect, useState } from "react";
+import ArticleCard from "./ArticleCard";
+
+const Articles = () => {
+    const [articles, setArticles] = useState([])
```

Comment:
Please include a 'loading' state and let the user know that the articles are still loading.